MAUI/TreeMap/getting-started.md

```diff
@@ -269,6 +269,8 @@ public class PopulationViewModel
 
 ![getting-started-with-maui-tree-map](images/getting-started/getting-started-with-maui-tree-map.png)
```

Comment:
Share PR in dev. Once ensured in staging and then will move to live site.

Reply:
Created dev PR - https://github.com/syncfusion-content/maui-docs/pull/2192